packages/cli/src/commands/ci/config/unset.ts

```diff
@@ -0,0 +1,47 @@
+import {Command, flags} from '@heroku-cli/command'
+import * as Heroku from '@heroku-cli/schema'
+import {Args, ux} from '@oclif/core'
+import {getPipeline} from '../../../lib/ci/pipelines'
+import {setPipelineConfigVars} from '../../../lib/api'
+
+const validateArgs = (argv: any) => {
```

Comment:
Just remembered this is a duplicate of what is in ci:config:set that was merged yesterday. We can consolidate them into a lib file. Name it `validateArgvPresent` or some such.